Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -741,6 +751,16 @@ final class LayerTests: XCTestCase {
             [11.077844 , 12.092919 ,  3.0350027,  4.7778125,  8.969137],
             accuracy: 1e-5)
     }
+    
+    func testLayerNormInference() {
+        Context.local.learningPhase = .inference
+        // This tests for a specific failure that had impacted the Transformer model.
+        let transformerTensor = Tensor<Float>(randomUniform: [1, 1, 768])
+        let transformerLayerNorm = LayerNorm(
+            featureCount: 768, axis: -1, epsilon: Tensor<Float>(1e-5))
```

Comment:
Minor suggestion just so it matches the style of the batch norm test.
```suggestion
            featureCount: 768,
            axis: -1,
            epsilon: Tensor<Float>(1e-5))
```